src/vorta/utils.py

```diff
@@ -318,3 +320,35 @@ def is_system_tray_available():
         is_available = tray.isSystemTrayAvailable()
 
     return is_available
+
+
+def validate_passwords(firstPass, secondPass):
+    passEqual = firstPass == secondPass
+    passLong = len(firstPass) > 8
+
+    if not passLong and not passEqual:
+        return trans_late('utils', "Passwords must be identical and greater than 8 characters long.")
+    if not passEqual:
+        return trans_late('utils', "Passwords must be identical.")
+    if not passLong:
+        return trans_late('utils', "Passwords must be greater than 8 characters long.")
+
+    return ""
+
+
+def password_transparency(encryption):
+    if encryption != 'none':
+        keyringClass = get_keyring().__class__.__name__
+        messages = {
```

Comment:
How about moving the Keyring name into the respective Keyring class instead of having a mapping here? The Keyring name itself wouldn't need translation, since it's common to use the English name.

Reply:
At least for macOS the names are translated https://support.apple.com/fr-fr/guide/keychain-access/kyca1083/mac

Reply:
I prefer having it centralized in one location, instead of importing the translations in all the keyrings.

Reply:
Also possible. Then keep the English name as attribute of each Keyring class and translate it where you need it. No importing of translations in every Keyring.

Reply:
That still separates the messages into each keyring, when it really shouldn't be there.

Reply:
Do translations even work with variable strings?

Reply:
`translate('utils', keyring.message)`
in VortaDBKeyring:
`message = "Saving the password to disk. To store password more securely install a supported secret store such as KeepassXC"`

Would that import `Saving the password to disk. To store password more securely install a supported secret store such as KeepassXC` to transifex?

Reply:
I don't want to break the translations, and doing this seems like the easiest way.

Reply:
A quick check of translations shows that translating variable strings does not work sadly. Have to go with the dictionary or the import in every keyring.
![Screenshot_20201201_020459](https://user-images.githubusercontent.com/67625312/100713252-a22f4b80-3379-11eb-995c-dd5635485456.png)